packages/instantsearch-react/examples/Search.js

```diff
@@ -75,7 +75,12 @@ class Search extends Component {
             defaultSelectedIndex="instant_search"
           />
           <HitsPerPage items={[10, 20, 30]} defaultHitsPerPage={20} />
-          <Range attributeName="price" />
+          <Range
+            attributeName="price"
+            translations={{
```

Comment:
The [`translatable`](https://github.com/algolia/instantsearch.js/blob/instantsearch-react/packages/instantsearch-react/src/translatable.js) HOC consumes a `translations` prop that is a map of translation key to translation strings/functions, and forwards a `translate` prop that is a function.